tests/test_service.py

```diff
@@ -341,6 +341,22 @@ def test_add_item_to_wishlist_bad_request_2(self):
                          .format(item.wishlist_id,
                                  test_wishlist.id))
 
+    def test_delete_wishlist(self):
+        """ Delete a Wishlist """
+        test_wishlist = self._create_wishlists(1)[0]
+        resp = self.app.delete(
+            "/wishlists/{}".format(test_wishlist.id), 
+	    content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_204_NO_CONTENT)
+        self.assertEqual(len(resp.data), 0)
+        # make sure they are deleted
+        resp = self.app.get(
+            "/wishlists/{}".format(test_wishlist.id), 
+            content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
```

Comment:
This test should fail, with 404

Reply:
![image](https://user-images.githubusercontent.com/41193680/96350858-c95de200-1085-11eb-8458-05f70e4cd05a.png)
This test passes now

Reply:
Maybe I am missing something here. Sorry for the confusion. 

The test case does a app.delete, it should be app.post().

Also, maybe send a message in response saying this wishlist is deleted, and then check for that message rather than len(), makes is full proof.

Then later confirms with the check for 404. 

We can also add a test case for deleting an existing item.

Reply:
@xiafanzeng - Added the test case and fixed the delete method.